datafusion/common/src/config.rs

```diff
@@ -704,6 +704,10 @@ config_namespace! {
 
         /// When set to true, the explain statement will print schema information
         pub show_schema: bool, default = false
+
+        /// Display format of explain. Default is "indent".
+        /// When set to "pretty", it will print the plan in a tree-rendered format.
```

Comment:
rather than `"pretty"` that prints out a tree I suggest we change the option name to `"tree"`